index-live.md

```diff
@@ -148,4 +148,9 @@ margin: auto;
       <p class="textcard">Virtual Desktop environment</p>
     </a>
 
+    <a class="mcard" href="https://live.usegalaxy.eu/?tool_id=interactivetool_climate_notebook" target="_top">
+      <img class="card-img-top" src="/assets/media/interactive/jp-climate.png" title="Jupyter Lab with Python ecosystem for ocean/atmosphere/land/climate science. The list of packages used in this image is similar to what is available for Pangeo" />
```

Comment:
the image is missing in this PR. You need to add it to assets/media/interactive/